_posts/2018-12-23-Release-3.3.0.md

```diff
@@ -0,0 +1,42 @@
+---
+author: Polybar Team
+account: polybar
+title: Polybar 3.3.0 Release
+excerpt: We added pseudo-transparency and fixed a bunch of bugs
+---
+
+# It's finally here
```

Comment:
Would you insist on using `h1` here? It is already used for the main post title so I thought it would be better to avoid its usage in the post body itself. Sorry to nitpick, Merry Christmas!

Reply:
Doesn't really matter to me. I just wanted to have some title at the beginning because the post looks kind of weird with only the header and then directly some text.

Reply:
Yeah, I agree, it certainly was something I was uncertain about when designing the header thing.